Pipelines/Templates/Package.yaml

```diff
@@ -12,9 +12,10 @@ jobs:
     demands: Unity2020LTS
   variables:
     unityLocation: ''
-    PackageVersion: '$(ProductVersion)-prerelease.$(Build.SourceBranchName).$(Build.BuildNumber)'
     ${{ if eq(variables['Build.SourceBranchName'], variables['SigningBranch']) }}:
       PackageVersion: $(ProductVersion)
+    ${{ else }}:

```

Comment:
Just curious if this actually changes anything functionally.  It seems like setting "-prerelease" string and then overriding if SigningBranch is set accomplishes the same thing without an else clause.

Reply:
It doesn't change anything from what we thought was working. :) Previous syntax gives me parsing error for redfinition of PackageVersion. I have no idea why previous builds didn't balk, I dug into it a little bit but didn't really see a clear answer. This syntax gets rid of the yaml parsing error and works as expected.